Editor/Deployment/DeploymentSettings.cs

```diff
@@ -222,29 +230,16 @@ public void Restore()
             ScenarioIndex = 1; // Selects "Single-Region Fleet" Deployment Scenario by default
             BuildFilePath = null;
             BuildFolderPath = null;
-            GetSettingResponse response = _coreApi.GetSetting(SettingsKeys.DeploymentGameName);
-            GameName = response.Success ? response.Value : null;
-            GetSettingResponse scenarioIndexResponse = _coreApi.GetSetting(SettingsKeys.DeploymentScenarioIndex);
-
-            if (scenarioIndexResponse.Success)
-            {
-                int? index = SettingsFormatter.ParseInt(scenarioIndexResponse.Value);
-                ScenarioIndex = index ?? 0;
-            }
-
-            GetSettingResponse serverPathResponse = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFolderPath);
-
-            if (serverPathResponse.Success)
-            {
-                BuildFolderPath = serverPathResponse.Value;
-            }
-
-            GetSettingResponse serverExePathResponse = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFilePath);
-
-            if (serverExePathResponse.Success)
-            {
-                BuildFilePath = serverExePathResponse.Value;
-            }
+            
+            var index = SettingsFormatter.ParseInt(_coreApi.GetSetting(SettingsKeys.DeploymentScenarioIndex).Value);
+            ScenarioIndex = index ?? 1;
```

Comment:
We should be storing the selected scenario as an enum itself and avoid having to do such conversions to Integers or indices

Reply:
done